### DISCUSSION TITLE
Cannot install AzuraCast via Proxmox GUI web panel

### DISCUSSION THREAD

**Discussion by gwbs** [AzuraCast/AzuraCast#6299] [Q&A]
I tried to install AzuraCast through the Proxmox GUI panel. I created a container to install it in. However, I am not able to use the curl command to install it. I am not able to install curl to the container. Furthermore, there is an error responding to debian.org via FTP. What is the solution to those problems?

**Comment by BusterNeece** [comment]
@gwbs I'd imagine there are two ways around this: either to have Proxmox build a VM instead, where you then follow our normal installation instructions (putting Docker in that VM, and then running AzuraCast in Docker)...the alternative would be to directly use the `azuracast/azuracast` image as your container image in Proxmox, though this wouldn't be as well-supported.

**Reply by gwbs** [reply]
So I presently have a Proxmox build on VirtualBox and installed docker and AzuraCast from there. After installation, I received an IP address to access AzuraCast from a web browser. However, I can't gain access to the web interface because the IP address assigned to me timed out.

**Reply by BusterNeece** [reply]
When our installer finishes, it refers to your public-facing IP (as seen by someone else connecting to you from outside your network); if you use something like Virtualbox or Proxmox, that IP will be something different that you can usually see via the respective control panel of the software.

**Reply by gwbs** [reply]
I installed Proxmox inside VirtualBox. Can I use SSH in VirtualBox to find that IP address by typing in ip addr on the command line to access the AzuraCast web interface?

**Reply by gwbs** [reply]
I am using the Promox Virtual Environment 7.4-3 with the GUI interface and setting up a container for Azuracast. Under the networking tab of my container, what should be my IP address? What is the gateway address? Should I leave the firewall on or turn it off? Under the DNS settings, should I leave that up to the host? I want to make sure after I install AzuraCast that I am able to access the web panel. Also, what IP should I be using after installation is completed? I know I have many questions. I am new to AzuraCast so I want to learn to get this installation right.

**Comment by Moonbase59** [comment]
Personally, I think running containers within a container within a hypervisor within a virtual machine manager within a host operating system can pose lots of unexpected problems and inefficiency, and requires a good working knowledge of networking and all components involved.

If anyhow possible, I’d instead try to run AzuraCast (dockerized) in a server OS (like Ubuntu 22.04 LTS Server) VM (not an LXC, for extra hardening and other options not available in an LXC), either using

- a bare-metal Proxmox, or
- a VirtualBox VM (might be advisable to use network bridging here, no NAT).

Having evaluated AzuraCast on both configurations, I can say you can get by with 2 CPUs, 4 GB RAM, and a 40 GB disk, and it runs rock-solid. Since Meilisearch was decommissioned, average CPU load was also reduced dramatically (load averages ~1.2–1.4 before, ~0.25 to 0.5 now).

Sorry not being able to help further, there are simply too many variables in a setup like you describe.
